File: Javascript/How good are you really?.js
// https://www.codewars.com/kata/how-good-are-you-really/train/javascript

function betterThanAverage(classPoints, yourPoints) {
    const average = classPoints.reduce((r, e) => {return r+=e}, 0) / classPoints.length
    return yourPoints > average 
  }
  

File: C#/How good are you really?.cs
// https://www.codewars.com/kata/how-good-are-you-really/train/csharp

using System.Linq;

public class Kata
{
  public static bool BetterThanAverage(int[] ClassPoints, int YourPoints)
  {
    return YourPoints > ClassPoints.Sum() / ClassPoints.Length;
  }
}